azure/v25.0.0/release.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: azure-25.0.0
+spec:
+  apps:
+  - name: azuredisk-csi-driver
+    version: 1.26.2-gs6
+    dependsOn:
+    - azure-cloud-controller-manager
+    - azure-cloud-node-manager
+  - name: azurefile-csi-driver
+    version: 1.30.2-gs1
+    dependsOn:
+    - azure-cloud-controller-manager
+    - azure-cloud-node-manager
+  - name: azure-cloud-controller-manager
+    version: 1.24.18-gs6
+    dependsOn:
+    - vertical-pod-autoscaler-crd
+  - name: azure-cloud-node-manager
+    version: 1.24.18-gs6
+    dependsOn:
+    - vertical-pod-autoscaler-crd
+  - name: capi-node-labeler
+    version: 0.5.0
+  - name: cert-exporter
+    version: 2.9.0
+    dependsOn:
+    - kyverno
+  - name: cert-manager
+    version: 3.7.6
+    dependsOn:
+    - prometheus-operator-crd
+  - name: chart-operator-extensions
+    version: 1.1.2
+    dependsOn:
+    - prometheus-operator-crd
+  - name: cilium
+    version: 0.24.0
```

Comment:
Sorry to chime in late, but can we do 0.25.1? It contains a fix for a bpf map size regression that can cause cilium to stop working on bigger clusters.

Reply:
I agree, otherwise we'll have to do a v25.1 release for azure too.
@njuettner wdyt?

Reply:
Sure no problem, I will update it 👍🏻